src/js/disability-benefits/utils/helpers.js

```diff
@@ -59,37 +59,51 @@ export function getUserPhase(phase) {
   return phase - 3;
 }
 
+export function getItemDate(item) {
+  if (item.receivedDate) {
+    return item.receivedDate;
+  } else if (item.documents && item.documents.length) {
+    return item.documents[item.documents.length - 1].uploadDate;
+  } else if (item.type === 'other_documents_list' && item.uploadDate) {
+    return item.uploadDate;
+  }
```

Comment:
This function is mainly to grab the `uploadDate` for files, which isn't in your list. We might be able to simplify this if the `uploadDate` was used to populate `date`, though.